nservicebus/pipeline/satellites.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Migrating Satellites
+summary: Extension point for handling messages
+tags: 
+related:
+ - nservicebus/pipeline/customizing-v6
+ - nservicebus/pipeline/features
+redirects:
+---
+
+In NServiceBus Versions 5 and below, Satellites were extension points to handle messages that arrive in different queues other than the endpoint's main queue. This extension point came in handy when the robustness of a separate queue was needed without having to create and setup a brand new Endpoint and configure any needed message mappings.
+
+These extensions were available via the `ISatellite` and `IAdvancedSatellite` interfaces.
+
+Starting from Versions 6 and above, these interfaces are no longer available. However, the same functionality can be achieved using the more advanced `Feature` extension. 
+
+
+## Migrating ISatellite and IAdvancedSatellite to Pipeline Features
+
+In Versions 6 and above, to get the satellite processing behavior, create a new feature. For more details, please read this article [to create a new feature and enabling it](/nservicebus/pipeline/features.md#feature-api).
+
+
+### Configuring the Satellite feature
+
+Once the feature is created, it needs to be configured using a message pipeline behavior.
+
+In the example below, the satellite input queue is called "targetqueue." 
+       
+snippet: SatelliteFeatureSetup
+
+The satellite behavior class should inherit from `PipelineTerminator<ISatelliteProcessingContext>`. This class handles the steps of what should occur when a message arrives in the satellite queue. 
```

Comment:
> This class handles the steps of what should occur when a message arrives in the satellite queue.

This doesn't read very well then, Maybe something like

ISatelliteProcessingContext provides the Terminate method where post-processing step are run when a message arrives in the satellite queue.